Archive/Wiedijk100Theorems/AscendingDescendingSequences.lean

```diff
@@ -138,15 +138,15 @@ theorem erdos_szekeres {r s n : ℕ} {f : Fin n → α} (hn : r * s < n) (hf : I
   -- Now that we have uniqueness of each label, it remains to do some counting to finish off.
   -- Suppose all the labels are small.
   by_contra q
-  push_neg at q 
+  simp (config := { zeta := false, proj := false }) only [not_exists, not_or, Nat.not_lt] at q
```

Comment:
This needs a porting note quoting the original tactic, and noting a regression in `push_neg`.